.github/workflows/build_and_test.yml

```diff
@@ -55,7 +55,7 @@ jobs:
         cd ros2_java_ws
         curl -sL file://${{ github.workspace }}/ros2_java_android.repos | vcs import src
         # Use checked out version of ros2_java
-        rm -rf src/ros2_java/ros2_java
+        rm -rf src/ros2-java/ros2_java
         ln --symbolic ${{ github.workspace }} src/ros2_java
```

Comment:
why the change, it doesn't same to match the symbolik link done next line

Reply:
I changed the repos file so that the path to the cloned repository matches the GitHub org where it came from (ie. ros2-java vs. ros2_java). Though it probably doesn't make a difference, I'll update the symlink for consistency.